content/en/docs/chart_template_guide/getting_started.md

```diff
@@ -117,6 +117,8 @@ NAME: full-coral
 LAST DEPLOYED: Tue Nov  1 17:36:01 2016
 NAMESPACE: default
 STATUS: DEPLOYED
+REVISION: 1
+TEST SUITE: None
```

Comment:
Resources are no longer output in Helm 3.  Output is no as follows:
```console
$ helm install full-coral ./mychart
NAME: full-coral
LAST DEPLOYED: Mon Mar  2 10:09:29 2020
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None

```

The following sentence is no longer valid : `In the output above, we can see that our ConfigMap was created. `